.changes/unreleased/🐛 Bug Fix-20221212-155748.yaml

```diff
@@ -0,0 +1,8 @@
+kind: "\U0001F41B Bug Fix"
+body: Remove required name flag while using data flag
```

Comment:
Can you be more clear? This is meant for combustible to users and developers. This needs to be understood without looking at the PR.